aten/src/ATen/native/cuda/Assert.cuh

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <c10/cuda/CUDAAssert.h>
```

Comment:
This file could also use some high level comments explaining what it does to uninformed readers (it's pretty clear to me, but I also know what you're doing...)